apps/dialtone-documentation/docs/about/whats-new/posts/2024-4-11.md

```diff
@@ -0,0 +1,282 @@
+dist/lib/attachment-carousel.cjs                   5.90 kB │ gzip: 1.84 kB │ map: 13.76 kB
+dist/lib/combobox-with-popover.cjs                 6.21 kB │ gzip: 2.35 kB │ map: 15.21 kB
+dist/chunks/index-2jPosQBn.js                      6.81 kB │ gzip: 2.26 kB │ map: 15.96 kB
+dist/lib/skeleton.cjs                              7.24 kB │ gzip: 2.10 kB │ map: 23.48 kB
+dist/lib/general-row.cjs                           8.41 kB │ gzip: 2.87 kB │ map: 18.36 kB
+dist/lib/input.cjs                                 8.62 kB │ gzip: 2.62 kB │ map: 22.12 kB
+dist/lib/combobox-multi-select.cjs                 9.04 kB │ gzip: 3.07 kB │ map: 22.41 kB
+dist/lib/message-input.cjs                        11.55 kB │ gzip: 3.59 kB │ map: 22.70 kB
+dist/lib/datepicker.cjs                           13.13 kB │ gzip: 3.47 kB │ map: 34.18 kB
+dist/lib/rich-text-editor.cjs                     14.56 kB │ gzip: 5.17 kB │ map: 52.99 kB
+dist/dialtone-vue.cjs                             14.98 kB │ gzip: 3.94 kB │ map:  0.10 kB
+dist/lib/editor.cjs                               14.98 kB │ gzip: 4.15 kB │ map: 28.14 kB
+dist/lib/popover.cjs                              14.98 kB │ gzip: 4.58 kB │ map: 41.03 kB
+dist/lib/emoji-picker.cjs                         23.10 kB │ gzip: 6.21 kB │ map: 50.83 kB
+✓ built in 2.15s
+```
+
+## Migration to dialtone >=9.27.1
+
+Replace any instances of the old imports with the corresponding new imports.
+Refer to the examples table below for clarity.
+
+| Previous 👎                                                                      | New 👍                                                                          | Enable Tree shaking (best) ✨                                                        |
+|----------------------------------------------------------------------------------|---------------------------------------------------------------------------------|-------------------------------------------------------------------------------------|
+| import { DtEmoji } from '@dialpad/dialtone/vue/emoji'                            | import { DtEmoji } from '@dialpad/dialtone/vue'                                 | import { DtEmoji } from '@dialpad/dialtone/vue/lib/emoji.js'                        |
+| import { DtMessageInput } from '@dialpad/dialtone/vue/message_input'             | import { DtMessageInput } from '@dialpad/dialtone/vue'                          | import { DtMessageInput } from '@dialpad/dialtone/vue/lib/message-input.js'         |
+| import { DtTooltipDirective } from '@dialpad/dialtone/vue/directives'            | import { DtTooltipDirective } from '@dialpad/dialtone/vue'                      | import { DtTooltipDirective } from '@dialpad/dialtone/vue/lib/tooltip-directive.js' |
+| import SpotBlankSpace from '@dialpad/dialtone/dist/css/vue/spot/SpotBlankSpace'; | import SpotBlankSpace from '@dialpad/dialtone/css/vue/spot/SpotBlankSpace.vue'; | -                                                                                   |
```

Comment:
seems to overlap the TOC a bit.

![Screenshot 2024-04-11 at 12 38 23 PM](https://github.com/dialpad/dialtone/assets/64808812/f5e0f2b3-9743-4ece-8a1a-1463ca9efee1)

Reply:
Thanks for catching that, fixing